### PR TITLE
Call for Content: 2020/09

### PR DESCRIPTION
**Pull request by domenkozar**
Hi all,

please add Nix news item in form of:

```markdown
- [Example news item title](http://example.com)

  Optional item description.

```

preferably as a PR against `master` branch (or a comment below).

Thank you <3

### CODE REVIEW

File: content/2020/08-nixos-weekly-2020-08.md [hunk 1]
```diff
@@ -91,7 +91,7 @@ Description:
 
 - [swift2nix: Run Swift inside Nix builds](https://euandre.org/2020/10/05/swift2nix-run-swift-inside-nix-builds.html)
 
-- [cargo2nix: Dramatically simpler Rust inside Nix](https://euandre.org/2020/10/05/cargo2nix-dramatically-simpler-rust-in-nix.htmla)
+- [cargo2nix: Dramatically simpler Rust inside Nix](https://euandre.org/2020/10/05/cargo2nix-dramatically-simpler-rust-in-nix.html)
```
Comment (Mic92) [inline content/2020/08-nixos-weekly-2020-08.md:94]: I suppose we should cherry pick this to `production`?

**Comment by domenkozar** [comment]
- [Eight Months of NixOS](https://catgirl.ai/log/nixos-experience/)

- [One week of NixOS](https://jae.moe/blog/2020/11/one-week-of-nixos/)

- [Moving into NixOS](https://foo-dogsquared.github.io/blog/posts/moving-into-nixos/)

**Comment by domenkozar** [comment]
- [Deploying NixOS using Terraform](https://nix.dev/tutorials/deploying-nixos-using-terraform.html)

- [Cachix: write access control for binary caches](https://blog.cachix.org/posts/2020-11-09-write-access-control-for-binary-caches/)

**Comment by domenkozar** [comment]
https://christine.website/blog/prometheus-grafana-loki-nixos-2020-11-20

http://www.haskellforall.com/2020/11/how-to-use-nixos-for-lightweight.html

https://www.srid.ca/i3-setup.html

https://www.srid.ca/2012301.html

https://gvolpe.com/blog/xmonad-polybar-nixos/

https://discourse.nixos.org/t/21-03-call-for-release-manager/10204

**Comment by FRidh** [comment]
For those that don't follow via discourse.

- Python 2.7 packages are no longer built https://discourse.nixos.org/t/python-2-7-and-3-9-and-hydra-builds
- Haskell packages now follow nightly https://discourse.nixos.org/t/new-ghc-default-version-8-10-2-package-versions-now-from-stackage-nightly
- Mach-nix now having conda packages as back-end https://discourse.nixos.org/t/mach-nix-create-python-environments-quick-and-easy/6858/86

**Comment by danielfullmer** [comment]
Running NixOS on Pinecube: https://github.com/danielfullmer/pinecube-nixos

**Comment by NaeosPsy** [comment]
Our New Nix Deployment Tool: deploy-rs 

https://serokell.io/blog/deploy-rs

**Pull request merged**